### ISSUE TITLE
error importing v3.0.7

### ISSUE THREAD

**Issue by jGaboardi** [pysal/libpysal#100]
```python
In [1]: import libpysal
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-b416346c1a00> in <module>()
----> 1 import libpysal

~/libpysal/libpysal/__init__.py in <module>()
     26 """
     27 from . import cg
---> 28 from . import io
     29 from . import weights
     30 from . import examples

~/libpysal/libpysal/io/__init__.py in <module>()
      1 from . import fileio
      2 from .tables import *
----> 3 from .iohandlers import *
      4 from .util import *
      5 open = fileio.FileIO

ModuleNotFoundError: No module named 'libpysal.io.iohandlers'
```

**Comment by sjsrey** [comment]
I can't reproduce this locally. The module is in [master](https://github.com/pysal/libpysal/tree/master/libpysal/io)?

**Comment by jGaboardi** [comment]
@sjsrey it appears to be something wrong with my cache, but I am not able to track it down.

my remote is synced with the upstream master and `git` says my local master up-to-date, but it clearly isn't because the directory structure is from the previous `libpysal` version:

```
(py3_lps) cpu:libpysal:io my_cpu$ ls
IOHandlers	__init__.py	__pycache__	fileio.py	geotable	tables.py	tests		util
```

Should I rebase?

I will close the issue for now due to it being on my end.

**Comment by sjsrey** [comment]
@jGaboardi that could be the culprit.

**Comment by jGaboardi** [comment]
@sjsrey I am at a loss. Have tried force resetting, force pulling, etc. with no affect, for example:

```
(py3_lps) my_cpu:io my_cpu$ git reset --hard HEAD
HEAD is now at ea2d44d Merge pull request #98 from sjsrey/lower
(py3_lps) my_cpu:io my_cpu$ git clean -f -d
(py3_lps) my_cpui:io my_cpu$ git pull
Already up-to-date.
(py3_lps) my_cpu:io my_cpu$ ls
IOHandlers	__init__.py	__pycache__	fileio.py	geotable	tables.py	tests		util
```

My local repo refuses to change. According to `git` everything is up-to-date on my local master, but it simply is not...

Next step is delete and re-clone?

**Comment by sjsrey** [comment]
maybe try: ```git pull --rebase upstream master```

**Comment by jGaboardi** [comment]
already tried that and just now tried it again. `git` still says i am up-to-date with `ea2d44d Merge pull request #98 from sjsrey/lower` but this in not reflected in my directory structure. tomorrow morning I am just going to remove the repo and start with a fresh `pysal/libpysal`.

**Comment by sjsrey** [comment]
another thing to check, from a shell:

```export PYTHONPATH```

maybe you have a path set that is pulling libpysal from elsewhere?

**Comment by jGaboardi** [comment]
@sjsrey I finally got the correct master to be recognized locally, but I can't say that I figured out what was actually happening. That much is still a bit of a mystery.

How I got it to work:

- deleted all files and subdirectories within the upper level `libpysal` directory, but not the upper level directory itself.

- then:
```
$ git reset --hard HEAD
$ git clean -f -d
$ git pull
```

It's worth noting that before actually deleting all the files in my `local master` I had tried all combinations of `merge`/`rebase`/`clean`/`hard reset` from both the `origin master ` and `upstream master` with absolutely no affect...
